### PR TITLE
feat: CodeArtifact Migration - Phase 1 & Phase 2

### PR DESCRIPTION
**Pull request by svanherk**
Phase 1:
- [x] Publish to npm
  - Change the name to the new name (found in spreadsheet)
  - Add `publishConfig` and `files` to `package.json`
  - Update the `release.yml` workflow accordingly (will vary depending on release action used)
- [x] Update README
  - Update install instructions and paths if name changed
  - Update `Releases` section to mention it publishes to `npm`

Phase 2:
- [x] Update all workflows to Node 16

Phase 2 only has this one step because it doesn't pull in any other repos we're updating.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
 > Building LitElement components? Use [BrightspaceUI/core](https://github.com/BrightspaceUI/core) instead.
 
-# d2l-localize-behavior
+# @brightspace-ui/localize-behavior
 
 [Polymer](https://www.polymer-project.org) mixin for localization of text, dates, times, numbers and file sizes. Also supports automatic language resolution, timezone and locale overrides.
 
-For further information on this and other components, refer to [The Brightspace UI Guide](https://github.com/BrightspaceUI/guide/wiki).
```
Comment (dlockhart) [inline README.md:7]: Oh, hello old friend!

File: README.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
-
 ## Installation
 
-`d2l-localize-behavior` can be installed from [Bower][bower-url]:
+Install from NPM:
 
 ```shell
-bower install d2l-localize-behavior
+npm install @brightspace-ui/localize-behavior
```
Comment (dlockhart) [inline README.md:12]: Another old friend!

**Review by dlockhart — COMMENTED** [review]
Not sure what that CI error is about, but it may be worth moving to the new test runner to avoid investigating that!

**Comment by svanherk** [comment]
> Not sure what that CI error is about, but it may be worth moving to the new test runner to avoid investigating that!

Yeah I _think_ it's same thing as `siren-sdk` with the path change, but I'm just gonna move to the new runner!

**Review by dlockhart — APPROVED** [review]

**Pull request merged**